store/CSDK/src/CarbonReader.h

```diff
@@ -35,6 +40,13 @@ class CarbonReader {
      */
     jobject carbonReaderObject;
 
+public:
+
+    /**
+     * jni env
+     */
+    JNIEnv *jniEnv;
+
     /**
      * create a CarbonReaderBuilder object for building carbonReader,
      * CarbonReaderBuilder object  can configure different parameter
```

Comment:
builder, projection, withHaaddopConf -- No need to have return value. They are not used. we can return void

Reply:
ok, build

Reply:
ok, done